backend/FwLite/FwDataMiniLcmBridge/Api/LcmHelpers.cs

```diff
@@ -83,6 +83,64 @@ internal static bool SearchValue(this ITsMultiString multiString, string value)
         '\u0640', // Arabic Tatweel
     ];
 
+    internal static MorphType FromLcmMorphTypeId(Guid? lcmMorphTypeId)
```

Comment:
as annoying as it is, we should probably have a full set of tests for both of these functions. It can be easily data driven, but if someone were to change one of these on accident it would probably go unnoticed for a long time.

Reply:
Added a note to #1850 to keep track of this.